apps/hash-frontend/src/graphql/queries/ontology/entity-type.queries.ts

```diff
@@ -57,7 +57,6 @@ export const getClosedMultiEntityTypesQuery = gql`
   query getClosedMultiEntityTypes(
     $entityTypeIds: [[VersionedUrl!]!]!
     $includeArchived: Boolean = false
-    $includeDrafts: Boolean = false
```

Comment:
Whops, forgot to mention this. It's removed as there are no drafts for entity types, yet.